packages/features/bookings/lib/handleSeats/reschedule/attendee/attendeeRescheduleSeatedBooking.ts

```diff
@@ -76,17 +77,19 @@ const attendeeRescheduleSeatedBooking = async (
     ]);
   }
 
-  const copyEvent = cloneDeep(evt);
-
-  const updateManager = await eventManager.reschedule(copyEvent, rescheduleUid, newTimeSlotBooking.id);
-
-  const results = updateManager.results;
+  // Add the new attendees to the new time slot booking attendees
+  for (const attendee of newTimeSlotBooking.attendees) {
+    const language = await getTranslation(attendee.locale ?? "en", "common");
+    evt.attendees.push({
+      email: attendee.email,
+      name: attendee.name,
+      language,
+    });
+  }
```

Comment:
The `evt` that was being passed from `handleNewBooking` only contained the rescheduling attendee. Let's add all attendees from the `newTimeSlotBooking` before updating the event.